Bootstrap_task/index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Bootstrap demo</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="./styles/styles.css" />
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-lg navbar-light bg-white">
+      <div class="container">
+        <a class="navbar-brand" href="#">nica reklama</a>
+        <button
+          class="navbar-toggler"
+          type="button"
+          data-bs-toggle="collapse"
+          data-bs-target="#navbarSupportedContent"
+          aria-controls="navbarSupportedContent"
+          aria-expanded="false"
+          aria-label="Toggle navigation"
+        >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div
+          class="collapse justify-content-end navbar-collapse"
+          id="navbarSupportedContent"
+        >
+          <ul class="navbar-nav mb-2 mb-lg-0">
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page" href="#"
+                >Pagrindinis</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Paslaugos</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Apie mus</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Straipsniai</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Kontaktai</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <div class="mainPage">
```

Comment:
if you are using BEM methodology, would be nice to keep it everywhere and do not mix with camel case

Reply:
Started page by tutorial video. so made same as on the video. After  that made everything by myself, thats why we came everything mixed. it was planned to change later.